{"name":"Story 991, Test 1205 ; ValidateCallTakeCantCancelCaseWithoutCancellationReason","status":"passed","statusDetails":{"known":false,"muted":false,"flaky":false},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1688056795798,"stop":1688056846300,"uuid":"66887fa3-3d67-4b00-b5f9-0e1f07e9e669","historyId":"c277972243833ab5285fe82598a6501","fullName":"Execute.CallTakerExecute.ValidateCallTakeCantCancelCaseWithoutCancellationReason","labels":[{"name":"package","value":"Execute.CallTakerExecute"},{"name":"testClass","value":"Execute.CallTakerExecute"},{"name":"testMethod","value":"ValidateCallTakeCantCancelCaseWithoutCancellationReason"},{"name":"parentSuite","value":"Default Suite"},{"name":"suite","value":"BCAA"},{"name":"subSuite","value":"Execute.CallTakerExecute"},{"name":"host","value":"LAP10-GUYL"},{"name":"thread","value":"31636@LAP10-GUYL.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"}],"links":[]}